spec/features/self_service/firms_index_spec.rb

```diff
@@ -105,6 +122,26 @@ def and_the_trading_names_section_is_showing_a_prompt_to_add_a_trading_name
     expect(firms_index_page).not_to have_trading_names
   end
 
+  def when_i_delete_the_first_trading_name
+    firms_index_page.trading_names.first.delete_button.click
+  end
+
+  def and_i_can_see_a_success_message
+    expected_message = "#{@first_trading_name_registered_name} has been successfully removed."
```

Comment:
Yeah, there are two sides to this. I tend go with having the message in the test so that it is tested.
If the source changes it's reasonable that the tests will need updated.

However the bang does sway me a little :-)